snuba/web/delete_query.py

```diff
@@ -147,7 +147,11 @@ def _delete_from_table(
         on_cluster=on_cluster,
         is_delete=True,
     )
-    _enforce_max_rows(query)
+    try:
+        _enforce_max_rows(query)
+    except NoRowsToDeleteException as e:
+        result: Result = {"extra_message": e.message}
```

Comment:
Also I don't think we need the `extra_message` here either I think you can make a `Result` object that just has empty data like `{"data": [], "meta": []}`, the reader result is kind of weird anyway because deleting data doesn't return how many rows you deleted or anything